egret/model_library/transmission/branch.py

```diff
@@ -409,10 +433,19 @@ def declare_eq_branch_loss_ptdf_approx(model, index_set, branches, bus_p_loads,
         branch = branches[branch_name]
         expr = 0
 
-        ptdf = branch['ldf']
-        for bus_name, coef in ptdf.items():
-            if ptdf_tol and abs(coef) < ptdf_tol:
-                coef = 0.
+        if branch['branch_type'] == 'transformer':
+            tau = branch['transformer_tap_ratio']
+            shift = math.radians(branch['transformer_phase_shift'])
+            g = tx_calc.calculate_conductance(branch)
+            #expr += (g/tau) * shift**2
```

Comment:
Should this also be uncommented?

Reply:
I uncommented it. I also fixed the nonzero reference angle issue in this PR.